### DISCUSSION TITLE
OSError: no library called "cairo" was found

### DISCUSSION THREAD

**Discussion by niuzhenjiang** [saleor/saleor#6497] [Q&A]
OSError: no library called "cairo" was found
platform:win10
python3.8.5
when i take python manage.py migrate,it's error for this:
OSError: no library called "cairo" was found
cannot load library 'C:\Program Files\GTK2-Runtime Win64\bin\libcairo-2.dll': error 0xc1
cannot load library 'libcairo.so': error 0x7e
cannot load library 'libcairo.2.dylib': error 0x7e
cannot load library 'libcairo-2.dll': error 0xc1
please help~~

**Comment by karolkielecki** [comment]
@niuzhenjiang Hey! What would you like to achieve? What's your version of Saleor? We need more info to reproduce the issue.
